here-naksha-lib-core/src/main/java/com/here/naksha/lib/core/util/storage/RequestHelper.java

```diff
@@ -111,6 +111,23 @@ public class RequestHelper {
     return new WriteFeatures<>(collectionName, List.of(updateOp));
   }
 
+  /**
+   * Helper method to create WriteFeatures request for updating multiple features.
+   *
+   * @param collectionName name of the storage collection
+   * @param features       feature object array to be updated
+   * @param <T>            any object extending XyzFeature
+   * @return WriteFeatures request that can be used against IStorage methods
+   */
+  public static @NotNull <T extends XyzFeature> WriteFeatures<T> updateFeaturesRequest(
```

Comment:
Depending on the clarity we get on the question I asked in group chat, this function may need to be converted to upsertFeaturesRequest (instead of update....) and so it will have to pass EWriteOp.UPSERT